bedrock/firefox/templates/firefox/nightly/whatsnew.html

```diff
@@ -65,7 +65,7 @@ <h4 class="mzp-c-callout-title">¿Quieres ayudar en España?</h4>
 
       <p>
           {% set attrs = 'href="#" class="nightly-experiments-link"' %}
-          {{ ftl('nightly-whatsnew-want-to-know-which-v2', attrs=attrs) }}
+          {{ ftl('nightly-whatsnew-want-to-know-which-v3', attrs=attrs) }}
```

Comment:
We should fall back to the old string for locales that have not yet updated
```suggestion
          {{ ftl('nightly-whatsnew-want-to-know-which-v3', fallback='nightly-whatsnew-want-to-know-which-v2', attrs=attrs) }}
```